example/acroname_power_supply_example.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Example Acronmae Power Module control logic.
+
+Right sequence to control power module:
+
+1. power_module.DiscoverAndConnectModule()
+2. Check no short circuit before configure and turn on
+  power_module.NoShortCircuit()
+3. power_module.ConfigurePowerSupply()
+   Note: default voltage and current limit are set based on the config.yaml
+4. power_module.TurnOnPowerSupply()
+5. Optionally, measure voltage and current:
+    power_module.GetVoltage()
+    power_module.GetCurrent()
+5. Optionally, you can change voltage and current limit:
+    power_module.ChangeVoltage(3300000)
+    power_module.ChangeCurrentLimit(2000000)
+6. Finally, once done with test, turn off and disconnect power module:
+    power_module.TurnOffAndDisconnect()
+
+Run with (your virtualenv must be activated first):
+python ./acroname_power_supply_example.py --config ./acroname_power_supply_example.yaml
+
+To log info into a file, add 2 flags following the command above:
+--log_file <LogFileName> --log_level info
+"""
+
+
+import tempfile
+import time
+
+#import example_plug
+import openhtf
+from openhtf.plugs.acroname import power_module_plug
+
+from openhtf.names import *
+
+class PowerModuleShortCircuitError(Exception):
+ """Short Circuit Found on Power Module."""
```

Comment:
This error sounds like it should be defined in the plug instead of in the test script, maybe.